packages/manager/apps/telecom/src/app/telecom/telephony/line/softphone/deleteDevice/delete-device.routing.js

```diff
@@ -2,22 +2,22 @@ export default /* @ngInject */ ($stateProvider) => {
   $stateProvider.state(
     'telecom.telephony.billingAccount.line.dashboard.softphone.delete',
     {
-      url: '/delete/:deviceId',
+      url: '/delete?deviceId',
```

Comment:
is it matching the design of the other pages of this app to use a query param instead of a path param ?

Reply:
In fact deviceId is optional params if is present so i remove only one device if isn't present so i remove all devices